Prace_domowe/Praca_domowa4/Grupa1/Adam_Frej/PD4.ipynb

```diff
@@ -0,0 +1,1479 @@
+{
```

Comment:
<p>W target encoding wyliczane są statystyki z danych (w tym z targetu), dlatego nie można dokonać tej transformacji a potem dzielić na zbiór treningowy i testowy (czy też wykonywać kroswalidację). Obecnie występuje data leak - korzystamy z targetu, którego w przypadku zbioru testowego algorytm nie powinien "widzieć".</p><p> </p><p>Należałoby zastosować Pipeline tak jak to było wyżej. Wówczas sklearn będzie automatycznie wykonywać target encoding po dokonaniu podziału na train/test w ramach kroswalidacji.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/240/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>